bitnami/grafana-operator/Chart.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: v2
```

Comment:
We're also standardizing the _Chart.yaml_ format by adding here the dependencies (already done) and also sorting alphabetically the fields and prettifying the format, you can do it automatically by running:
```
$ yq r --tojson Chart.yaml | jq . | yq r --prettyPrint - > Chart.yaml.new
$ mv Chart.yaml.new Chart.yaml
```